dataprofiler/data_readers/data_utils.py

```diff
@@ -17,8 +29,10 @@
 
 logger = dp_logging.get_child_logger(__name__)
 
+JSONType = Union[str, int, float, bool, None, List["JSONType"], Dict[str, "JSONType"]]
```

Comment:
just to make it a bit cleaner... let's move this to `_typing.py` https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/_typing.py

Reply:
awesome! Thx